build/zip.py

```diff
@@ -13,7 +13,9 @@
 PATHS_TO_SKIP = [
   'angledata', #Skipping because it is an output of //ui/gl that we don't need
   './libVkLayer_', #Skipping because these are outputs that we don't need
-  './VkLayerLayer_', #Skipping because these are outputs that we don't need
+  './libVkICD_mock_', #Skipping because these are outputs that we don't need
+  './VkLayer_', #Skipping because these are outputs that we don't need
```

Comment:
this is a new skip but i don't see any changes in the zip manifests related to it; is it still needed?

Reply:
VkLayerLayer was incorrect, fixing to VkLayer. This was handled by disabling building of the debug layers, however keeping it here just in case. I can remove the the entries for debug layers though.

Reply:
> keeping it here just in case

If it not used, maybe it can be safely deleted?

Reply:
removed

Reply:
@nornagon btw is there a reason why we don't have the zip manifests on Windows?